py/vtdb/vtgatev2.py

```diff
@@ -65,18 +38,32 @@ def handle_app_error(exc_args):
   return dbexceptions.DatabaseError(exc_args)
 
 
-@log_exception
-def convert_exception(exc, *args):
+def convert_exception(exc, *args, **kwargs):
   new_args = exc.args + args
+  if kwargs:
+    new_args += tuple(kwargs.values())
```

Comment:
nitpicky, but this could be kwargs.itervalues(), since you're making it a tuple anyways. It just needs to be iterated over.